doc/method/results.md

```diff
@@ -0,0 +1,342 @@
+
+# Results
+
+## Alignment
+
+The alignment procedure examines every source node.  The following is
+a breakdown of the disposition of source nodes, across all sources, as
```

Comment:
disposition = how it is disposed of (dictionary definition) after the "examination" mentioned in first sentence. explanations of the possible dispositions follow the table. would "what it decides (or determines) about each source node" be better?